File: AppGenerateFiles/helpdesk/FrontModel/ViewCalendarioByDependenciaViewModel.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
class ViewCalendarioByDependencia extends EntityClass {
   constructor(props) {
       super(props, 'ViewHelpdesk');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Case;
   /**@type {Number}*/ Id_TareaPadre;
   /**@type {Date}*/ Fecha_Inicio;
   /**@type {String}*/ Estado;
   /**@type {Number}*/ IdCalendario;
   /**@type {Number}*/ Id_Tarea;
   /**@type {Number}*/ Id_Dependencia;
}
export { ViewCalendarioByDependencia }

File: AppGenerateFiles/helpdesk/Model/ViewCalendarioByDependenciaViewModel.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class ViewCalendarioByDependencia : EntityClass {
       public int? Id_Case { get; set; }
       public int? Id_TareaPadre { get; set; }
       public DateTime? Fecha_Inicio { get; set; }
       public string? Estado { get; set; }
       public int? IdCalendario { get; set; }
       public int? Id_Tarea { get; set; }
       public int? Id_Dependencia { get; set; }
   }
}
